### PR TITLE
Add alias for whereNull / whereNotNull

### PR DESCRIPTION
**Pull request by JWaldecker**

### CODE REVIEW

File: src/main/java/org/javawebstack/orm/query/Query.java [hunk 1]
```diff
@@ -164,6 +164,14 @@ public Query(Repo<T> repo, Class<T> model) {
         where.notNull(left);
         return this;
     }
+    
+    public Query<T> whereNull(Object left) {
+        return where.isNull(left);
```
Comment (TimothyGillespie) [inline src/main/java/org/javawebstack/orm/query/Query.java:169]: I'm not very familiar with this part of the code but it seems to me it should be:
```suggestion
        where.isNull(left);
        return this;
```

File: src/main/java/org/javawebstack/orm/query/Query.java [hunk 2]
```diff
@@ -164,6 +164,14 @@ public Query(Repo<T> repo, Class<T> model) {
...
+    }
+    
+    public Query<T> whereNotNull(Object left) {
+        return where.notNull(left);
```
Comment (TimothyGillespie) [inline src/main/java/org/javawebstack/orm/query/Query.java:173]: Same here.

File: src/main/java/org/javawebstack/orm/query/Query.java (hunk 1 above)
Comment (x7airworker) [reply src/main/java/org/javawebstack/orm/query/Query.java:169]: This should work fine

Comment (JanHolger) [reply src/main/java/org/javawebstack/orm/query/Query.java:169]: His code is totally fine and smaller

**Pull request merged**